src/modules/dogs/dogs.store.js

```diff
@@ -0,0 +1,32 @@
+import { getDogsList } from "./services";
+
+export default {
+  namespaced: true,
+  state: {
+    dogsData: [],
+  },
+
+  getters: {
+    dogsDataGatters(state) {
```

Comment:
As this getter is returning data directly, you should be using mapState, getters should be used in case there is some sort of data processing. 
Also naming convection should be standard, it's a getter no need to write it specifically in naming. 
Eg: A getter which returns sorted list will be named sortedDogsData which will return a lambda, 
```js
(sortBy, order) => {
 // sort data
 return sortedDogsData
}
```

Reply:
plz check